Xwt.Gtk/Xwt.GtkBackend/GtkAlertDialog.cs

```diff
@@ -123,7 +123,7 @@ public GtkAlertDialog (ApplicationContext actx, MessageDescription message)
 			}
 			
 			if (message.AllowApplyToAll) {
-				Gtk.CheckButton check = new Gtk.CheckButton ("Apply to all");
+				Gtk.CheckButton check = new Gtk.CheckButton (Application.TranslationCatalog.GetString("Apply to all"));
```

Comment:
I have an error with conflict Xwt.Application with Gtk.Application. The line 126 should be:

Gtk.CheckButton check = new Gtk.CheckButton (Xwt.Application.TranslationCatalog.GetString("Apply to all"));

Adding an explicit reference to a Xwt.Application